### PR TITLE
Remove policies from vulnerability management

### PR DESCRIPTION
**Pull request by bradr5**
## Description

Remove policies page from vulnerability management

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Screenshot 2023-05-31 at 2 42 04 AM
Screenshot 2023-05-31 at 2 42 10 AM

Screenshot 2023-05-31 at 2 48 26 AM
Screenshot 2023-05-31 at 2 42 58 AM

Screenshot 2023-05-31 at 2 44 04 AM
Screenshot 2023-05-31 at 2 44 09 AM

Screenshot 2023-05-31 at 2 45 04 AM
Screenshot 2023-05-31 at 2 45 09 AM

Screenshot 2023-05-31 at 2 46 32 AM
Screenshot 2023-05-31 at 2 46 38 AM

### CODE REVIEW

File: ui/apps/platform/src/Containers/Workflow/WorkflowListPageLayout.js [hunk 1]
```diff
@@ -67,6 +67,9 @@ const WorkflowListPageLayout = ({ location }) => {
     const selectedRow = workflowState.getSelectedTableRow();
 
     const header = upperFirst(pluralize(entityLabels[pageListType]));
+    console.log('entityLabels', entityLabels);
+    console.log('pageListType', pageListType);
+    console.log('header', header);
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/Workflow/WorkflowListPageLayout.js:72]: These `console.log` cause style-checks to fail.

File: ui/apps/platform/src/utils/entityRelationships.ts [hunk 2]
```diff
@@ -26,7 +26,6 @@ export const useCaseEntityMap = {
         ...baseEntities,
     ],
     [useCaseTypes.VULN_MANAGEMENT]: [
-        entityTypes.POLICY,
```
Comment (pedrottimark) [inline ui/apps/platform/src/utils/entityRelationships.ts:29]: Good work to update Jest snapshots.

File: ui/apps/platform/src/Containers/VulnMgmt/List/Policies/VulnMgmtListPolicies.js [hunk 3]
```diff
@@ -1,358 +0,0 @@
...
-import entityTypes from 'constants/entityTypes';
-import { LIST_PAGE_SIZE } from 'constants/workflowPages.constants';
-import queryService from 'utils/queryService';
-import { workflowListPropTypes, workflowListDefaultProps } from 'constants/entityPageProps';
-import { actions as notificationActions } from 'reducers/notifications';
-import { deletePolicies } from 'services/PoliciesService';
-import removeEntityContextColumns from 'utils/tableUtils';
-import { policySortFields } from 'constants/sortFields';
-
-import { UNSCOPED_POLICY_LIST_FRAGMENT } from 'Containers/VulnMgmt/VulnMgmt.fragments';
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/VulnMgmt/List/Policies/VulnMgmtListPolicies.js:33]: It looks like this is the only reference to the GraphQL fragment.

**Comment by pedrottimark** [comment]
@bradr5 for your info, not a blocker to merge, here is semi-404 page if I visit /main/vulnerability-management/policies

vulnerability-management_policies

Do you want to investigate together how to render 404 page as for /main/vulnerability-management/unknown
vulnerability-management_unknown

File: ui/apps/platform/src/Containers/VulnMgmt/List/Policies/VulnMgmtListPolicies.js (hunk 3 above)
Comment (bradr5) [reply ui/apps/platform/src/Containers/VulnMgmt/List/Policies/VulnMgmtListPolicies.js:33]: Good catch 👍

**Comment by bradr5** [comment]
> @bradr5 for your info, not a blocker to merge, here is semi-404 page if I visit /main/vulnerability-management/policies
> 
> Do you want to investigate together how to render 404 page as for /main/vulnerability-management/unknown

I would like that. I was initially investigating that myself and found `resourceLabels` in `common.ts`, but realized we were handling `role` the same way. So I figured it wasn't something we were too worried about although I agree It would be better to render 404 for the whole page.

**Review by pedrottimark — APPROVED** [review]
Less is more!

**Review by vjwilson — APPROVED** [review]
I'm good with this, as long as Mandar agrees with the radical removal of all references to policies on all the pages.

File: ui/apps/platform/src/Containers/VulnMgmt/Entity/Cluster/VulnMgmtClusterOverview.js [hunk 4]
```diff
@@ -3,7 +3,6 @@ import React, { useContext } from 'react';
 import CollapsibleSection from 'Components/CollapsibleSection';
 import DateTimeField from 'Components/DateTimeField';
 import Metadata from 'Components/Metadata';
-import PolicyStatusIconText from 'Components/PatternFly/IconText/PolicyStatusIconText';
```
[the remark this answers is not in the document]
Comment (pedrottimark) [reply ui/apps/platform/src/Containers/VulnMgmt/Entity/Cluster/VulnMgmtClusterOverview.js:6]: @md2119 @vjwilson 
Here is a different way to ask the question above: does it matter whether or not we delete the policy count and status fields from GraphQL queries?

If I read these changes correctly, we deleted **rendering** of the information, but not **request** for the information.

[the remark this answers is not in the document]
Comment (md2119) [reply ui/apps/platform/src/Containers/VulnMgmt/Entity/Cluster/VulnMgmtClusterOverview.js:6]: @pedrottimark graphQL is official not supported for the customers. We do not provide guarantees if they choose to use graphQL. Nonetheless, the sections on the old VM will be deprecated starting 4.1.

**Pull request merged**